main.tf

```diff
@@ -0,0 +1,31 @@
+ module "consul" {
+   source = "./modules/consul"
+   count_consul = "${var.count_consul}"
+   version = "0.0.1"
```

Comment:
How do you handle versioning? As I remember this parameter is only for modules installed from a module registry, such as the Terraform Registry or Terraform Enterprise's private module registry.